src/common/board.rs

```diff
@@ -156,6 +172,20 @@ impl Board for FakeBoard {
             anyhow::bail!("could not find I2C with name {}", name)
         }
     }
+    fn get_pwm_duty(&self, pin: i32) -> anyhow::Result<f64> {
+        Ok(*self.pin_pwms.get(&pin).unwrap_or(&0.0))
```

Comment:
is it the behavior in RDK when a pin is not configured as a PWM? if so we don't need the Result then

Reply:
Is there any reason, at least for the Micro-RDK, we would want to know if the pwm was actually 0.0 or not there? Or do we just interpret 0.0 as not configured in any case?

Reply:
@npmenard yes this is the behavior in RDK I believe, I'll remove Result

@mattjperez you can probably distinguish by querying the frequency (if frequency is 0, then the PWM was not there), so I don't think we have to distinguish between the scenarios in this call